pyspark_datasources/robinhood.py

```
@@ -0,0 +1,253 @@
+        # Official Robinhood signature format: f"{api_key}{current_timestamp}{path}{method}{body}"
+        # For GET requests with no body, omit the body parameter
+        if method.upper() == "GET" and not body:
+            message_to_sign = f"{self.api_key}{timestamp}{path}{method.upper()}"
+        else:
+            message_to_sign = f"{self.api_key}{timestamp}{path}{method.upper()}{body}"
+            
+        signed = self.signing_key.sign(message_to_sign.encode("utf-8"))
+        signature = base64.b64encode(signed.signature).decode("utf-8")
+        return signature
+
+    def _make_authenticated_request(self, method: str, path: str, params: Dict = None, json_data: Dict = None):
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)